src/genegraph_ui/pages/documentation/views.cljs

```diff
@@ -24,7 +24,7 @@
        [:li [:a {:href (rfe/href :gene-validity)}"gene validity"]]]
       [:p.menu-label "explore"]
       [:ul.menu-list
-       [:li [:a "Genegraph GraphiQL" [:span.icon [:i.fas.fa-link]]]]]])
+       [:li [:a {:href (s/replace genegraph-ui.BACKEND_HTTP "api" "ide")  } "Genegraph GraphiQL" [:span.icon [:i.fas.fa-link]]]]]])
```

Comment:
We could have the link auto-fill the query from the current UI page too. If given a `query` url param, graphiql puts that as the contents of the query box.